tests/robustness/failpoint/gofail.go

```diff
@@ -59,6 +59,15 @@ var (
 	BeforeApplyOneConfChangeSleep            Failpoint = killAndGofailSleep{"beforeApplyOneConfChange", time.Second}
 	RaftBeforeSaveSleep                      Failpoint = gofailSleepAndDeactivate{"raftBeforeSave", time.Second}
 	RaftAfterSaveSleep                       Failpoint = gofailSleepAndDeactivate{"raftAfterSave", time.Second}
+
+	// AllowBatchCompactBeforeSetFinishedCompactPanic is used to trigger
+	// that compactBeforeSetFinishedCompact failpoint only if the current
+	// revision number is higher than that batch limit.
+	AllowBatchCompactBeforeSetFinishedCompactPanic Failpoint = goPanicFailpoint{
+		failpoint: "compactBeforeSetFinishedCompact",
+		trigger:   triggerCompact{multiBatchCompaction: true},
+		target:    AnyMember,
+	}
```

Comment:
```suggestion
	CompactBeforeSetFinishedCompactPanic Failpoint = goPanicFailpoint{"compactBeforeSetFinishedCompact", triggerCompact{multiBatchCompaction: true}, AnyMember}
```

Reply:
Hi @serathius , `CompactBeforeSetFinishedCompactPanic` is already existing at https://github.com/etcd-io/etcd/blob/57907746e39da97ed3ce2fbd1f97e8ceb2ce8199/tests/robustness/failpoint/gofail.go#L47 Do you mean that I should replace existing one?

I add new one here because this issue has a requirement that the compactor has to delete tombstone in one batch and update `UnsafeSetFinishedCompact` value in next round. IMO, it's better to have two go-failpoints here.

Reply:
We can have both, best to even duplicate all failpoints with triggerCompact to have option with and without multiBatchCompaction.